recipes/ler/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "ler" %}
+{% set version = "0.2.3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: ac75714d6d1d5da0c5726e6f07de984619e5b53282e32dc9a35fad4586a65fe6
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python >=3.10
+  run:
+    - astropy >=5.1
+    - bilby >=1.0.2
+    - gwsnr >=0.1
```

Comment:
It looks like `gwsnr` is not packaged on conda-forge. You'll have to submit a PR for that package first or add it to this PR. See:
https://conda-forge.org/docs/maintainer/adding_pkgs.html#adding-multiple-packages-at-once